src/DataTable/Table.tsx

```diff
@@ -141,6 +152,66 @@ const StyledTable = styled.table<React.ComponentPropsWithoutRef<'table'>>`
     text-align: start;
   }
 
+  /* TableCellSkeleton */
+  .TableCellSkeleton {
+    padding: 0;
+  }
+
+  .TableCellSkeletonItems {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .TableCellSkeletonItem {
```

Comment:
If we see this style of "loading" placeholder come up again, we should consider creating some kind of "SkeletonLoader" component that can be reused. We already use something identical in TreeView.